CHANGES.md

```diff
@@ -28,17 +28,11 @@ OpenSSL 3.3
 
 ### Changes between 3.2 and 3.3 [xx XXX xxxx]
 
- * In provider_conf_load, changed the soft_load setting to require a fixed set
-   of values to activate/deactivate (1/0, yes/no, true/false, on/off), in either
-   all lower or all uppercase.  All other values return an error.
-
-   *Neil Horman*
-
- * The activate configuration setting for providers in openssl.cnf has been
-   updated to require a value of [1|yes|true|on] (in lower or UPPER case) to
-   activate the provider.  Conversely a setting [0|no|false|off] will prevent
-   provider activation.  All other values, or the omission of a value for this
-   setting will result in an error.
+ * The activate and soft_load configuration setting for providers in
+   openssl.cnf has been updated to require a value of [1|yes|true|on]
+   (in lower or UPPER case) to enable the setting. Conversely a setting
```

Comment:
Might be better to say `a value of ... will disable the setting` to avoid repetition of the word `setting` (and singular here is fine - "a value")

Reply:
fixed